autoscale/src/main/java/com/sequenceiq/periscope/config/ServiceEndpointConfig.java

```diff
@@ -27,6 +27,15 @@
     @Value("${periscope.cloudbreak.url:}")
     private String cloudbreakUrl;
 
+    @Value("${periscope.freeipa.url:}")
+    private String freeIpaServerUrl;
+
+    @Value("${periscope.freeipa.serviceid:}")
+    private String freeIpaServiceId;
+
+    @Value("${periscope.freeipa.contextPath:}")
+    private String freeIpaContextPath;
+
```

Comment:
if sync is needed indeed, then relevant changes should be made in dps-k8s and cloud-services-infra repositories

Reply:
Ack, I think a PR has been merged for dps-k8s: https://github.infra.cloudera.com/thunderhead/dps-k8s/pull/234, will create one for CSI, if manual sync is required